agentops/client.py

```diff
@@ -15,12 +15,14 @@
 from typing import Optional, List
 import traceback
 from .log_config import logger, set_logging_level_info
+from decimal import *
```

Comment:
Don't `import *`. Generally bad practice, as it can have unintentional overrides. 

Better to just `from decimal import Decimal`